Week-3/3.1/index.js

```diff
@@ -0,0 +1,34 @@
+//memoization
+
+function memoize(fn) {
+    let cache = new Map();
+    return function (...args) {
+        let key = args.toString();
```

Comment:
Any edge cases missing? What if the list of args is null/empty, then there will be same key for all undefined params